server/README.md

```diff
@@ -110,6 +110,19 @@ This demo implementation rejects login attempts by the demo user.
 
 This demo implementation logs a message to the demo channel whenever a user logs in.
 
+## [user\_hooks.go](user_hooks.go)
+
+### UserHasBeenCreated
+
+This demo implementation logs a message to demo channel when a new user is created.
+
+## [file\_hooks.go](file_hooks.go)
+
+### FileWillBeUploaded
+
+This demo implementation logs a message to demo channel when a file is uploaded.
```

Comment:
`when` --> `whenever` as well

Reply:
Hi.How can I use this in my plugin.Please provide documentation with example related to this for beginners in Go :(

Reply:
Hi @00Spider47! This PR has an example here https://github.com/mattermost/mattermost-plugin-demo/pull/23/files#diff-2ce873f0219bcda5f21b4a9c890b1f01

We also have some documentation here to get started https://developers.mattermost.com/extend/plugins

Let me know if that's helpful?